authorizations/authorization-ranger/build.gradle.kts

```diff
@@ -126,7 +129,7 @@ tasks {
 
 tasks.test {
   doFirst {
-    environment("HADOOP_USER_NAME", "test")
+    environment("HADOOP_USER_NAME", "gravitino")
   }
```

Comment:
The `HADOOP_USER_NAME` default is `gravitino`, If we do not need to set a different value, we can remove this paragraph code.

Reply:
OK, reverted.

Reply:
Default HADOOP_HOME is anoymous. So I added it back.